target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-wtr-m2133hp.dts

```diff
@@ -0,0 +1,420 @@
+// SPDX-License-Identifier: ISC
+/*
+ * Copyright (c) 2015 - 2016, The Linux Foundation. All rights reserved.
```

Comment:
So, is this upstream somewhere?

Reply:
Yes. I started porting based on [qsdk qcom-ipq4019-ap.dk04.1.dtsi](https://source.codeaurora.org/quic/qsdk/oss/kernel/linux-msm/tree/arch/arm/boot/dts/qcom-ipq4019-ap.dk04.1.dtsi?h=NHSS.QSDK.12.0.r2).